sockeye/inference.py

```diff
@@ -1096,3 +1133,16 @@ def _get_best_from_beam(self,
             score = accumulated_scores[idx].asscalar()
             result.append(Translation(sequence, attention_matrix, score))
         return result
+
+    def print_beam(self, sequences, scores, finished, active, timestep):
```

Comment:
I think this function should either be removed or changed to use the logger. We never want to print something, imo.